src/components/sbb-navigation-section/sbb-navigation-section.tsx

```diff
@@ -275,23 +266,14 @@ export class SbbNavigationSection implements ComponentInterface {
 
   // Set focus on the first focusable element.
   private _setNavigationSectionFocus(): void {
```

Comment:
By removing the "trick" of setting the focus on the wrapper when `mostRecentModality !== 'keyboard'`, if I open the nav by click and press `Tab` I'll land on the second focusable element and not the first. This happens for both the navigation and the navigation section. Is this wanted?